packages/platform-core/src/cwd_package_json_reader.ts

```diff
@@ -0,0 +1,20 @@
+import * as path from 'path';
+import { PackageJson, PackageJsonReader } from '.';
+
+/**
+ * Loads the contents of package.json from process.cwd().
+ *
+ * Throws if no package.json is present
+ */
+export class CwdPackageJsonReader {
```

Comment:
I wonder if this should just be another method on PackageJsonReader. ie `.readFromCwd` or such?